src/Kafka/CreateKafkaInstance/components/InstanceInfo.tsx

```diff
@@ -53,126 +59,163 @@ export const InstanceInfo: VoidFunctionComponent<
   connectionRate,
   messageSize,
   onClickQuickStart,
+  streamingUnits,
 }) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation("create-kafka-instance");
 
   return (
     <TextContent>
-      <Text component={TextVariants.h3}>
-        {t("create-kafka-instance:instance_information")}
-      </Text>
-      <TextList component={TextListVariants.dl}>
-        {!isLoading && (
-          <Grid sm={6} lg={12} hasGutter>
-            {isTrial && (
-              <GridItem>
-                <TextListItem component={TextListItemVariants.dt}>
-                  {t("kafka:duration")}
-                </TextListItem>
-                <TextListItem component={TextListItemVariants.dd}>
-                  {t("create-kafka-instance:duration_value", {
-                    value: trialDurationInHours,
-                  })}
-                </TextListItem>
-              </GridItem>
+      <Card>
+        <CardHeader>
+          <Text component={TextVariants.h3}>{t("instance_information")}</Text>
+        </CardHeader>
```

Comment:
There are a few changes here that I think would be good to make:
* replace `<CardHeader>` with `<CardTitle component="h2">` and don't use the `<Text>` component. For example
  `<CardTitle component="h2">Details</CardTitle>`
  * With the modal title being an h1, these headings should really be h2. But continuing to use the `<TextContent>` would make the `<h2>` elements too big. 
  * This change would be applied here and in the next card.
* In the `<CardBody>`, replace the `<TextList>` component and children with a `<DescriptionList isCompact>` component with the children it uses. 
  * This buys us a little bit more space by being able to use the `isCompact` prop.

If all of these changes are made, then I don't think we need to wrap them in the `<TextContent>` component at all.